moz_kinto_publisher/test_publisher.py

```diff
@@ -0,0 +1,228 @@
+import main
+import unittest
+import hashlib
+from datetime import datetime, timezone
+
+
+def make_record(run_id, *, diff):
+    timestamp = main.timestamp_from_run_id(run_id)
+    record_type = "diff" if diff else "full"
+    record_time = timestamp.isoformat(timespec="seconds")
+    identifier = f"{record_time}Z-{record_type}"
+
+    random_id = hashlib.sha256(identifier.encode("utf-8")).hexdigest()
+
+    return {
+        "schema": 1,
+        "details": {"name": identifier},
+        "attachment": {
+            "hash": "hash",
+            "size": 0,
+            "filename": f"{run_id}",
+            "location": "abc",
+            "mimetype": "application/octet-stream",
+        },
+        "incremental": diff,
+        "id": random_id,
+        "last_modified": 1,
```

Comment:
Should this include a parent field if this is a diff?

Reply:
Absolutely should. Thanks!